bundle-src/webpack.config.js

```diff
@@ -102,6 +102,11 @@ function makeConfig(kind, name) {
       filename: `${name}.bundle.js`,
       uniqueName: name,
     },
+    resolve: {
+      alias: {
+        react: path.resolve('./node_modules/react')
```

Comment:
this is the bit that seems to be causing build problems on my end, will try and debug my issues now

Reply:
Yeah, I added that in because I somehow had two copies of React in the build... something's fishy with this webpack config

Reply:
ooookay weird...I guess it might be a your machine problem having two copies of react then...? idk

Reply:
lol yeah commenting it out fixes it for me, not sure about your two copies bug tho - wouldn't know where to look to see if I have it or not?!

Reply:
I think it's related to workspaces - `graphics-src` and `scores-src` both have an instance of React so somehow one is making it into the other...

Reply:
how do I tell if this is a bug for me too or not?

Reply:
If you can get the dashboards to display without React throwing an error, it's not affecting you.

Reply:
I mean seems to work for me off the bat if I just comment out the line (I also purged my node_modules and did another yarn, would it be worth u trying the same to eliminate weirdness?)

Reply:
Possibly, let me give it a whack

Reply:
Aha, think I figured it out - was likely caused by me running `yarn` both in the root _and_ in `bundle-src` - obliterating node_modules fixed it, so this should be unnecessary. Will remove this on the next push.